Source/UI/NewUI/CommonUI/View/Component/InputView.swift

```diff
@@ -204,7 +204,8 @@ extension InputView: TextFieldViewDelegate {
     }
 
     func textFieldShouldEndEditing(textField: UITextField, replacementString: String) -> Bool {
-        delegate?.textFieldShouldEndEditing(textField: textField, replacementString: replacementString) ?? true
+        textFieldContainer.layer.borderColor = style?.textfield.normalBorderColor.cgColor
+        return delegate?.textFieldShouldEndEditing(textField: textField, replacementString: replacementString) ?? true
```

Comment:
What if the delegate returns a false and doesn't let text field end editing? Not sure how that may happen but this code will change border color whilst keeping text field editing.

Wonder if this is more accurate:
```suggestion
        let shouldEndEditing = delegate?.textFieldShouldEndEditing(textField: textField, replacementString: replacementString) ?? true
        if shouldEndEditing {
            textFieldContainer.layer.borderColor = style?.textfield.normalBorderColor.cgColor
        }
        return shouldEndEditing
```